README.md

```diff
@@ -673,9 +673,19 @@ interface LayoutImage extends Node {
 type TableColumnSettings = {
 	hideOnMobile: boolean
 	sortable: boolean
-	sortType: 'text' | 'number' | 'date' | 'currency' | 'percent'
+	sortType?: 'text' | 'number' | 'date' | 'currency' | 'percent'
 }
 
+type TableChildren =
+  | [TableCaption, TableBody, TableFooter]
+  | [TableBody, TableFooter]
+  | [TableCaption, TableBody]
+  | [TableBody]
+  | [TableCaption, TableHeader, TableBody, TableFooter]
+  | [TableHeader, TableBody, TableFooter]
+  | [TableCaption, TableHeader, TableBody]
+  | [TableHeader, TableBody]
```

Comment:
Suggestion: This can be simplified (Pls remove comments)
```suggestion
  | [TableCaption, TableBody, TableFooter?] // It covers also [TableCaption, TableBody]
  | [TableBody, TableFooter?] // It covers also [TableBody]
  | [TableCaption, TableHeader, TableBody, TableFooter?] // It covers also  [TableCaption, TableHeader, TableBody]
  | [TableHeader, TableBody, TableFooter?] // It covers also [TableHeader, TableBody]
```